Update documentation for security ticketing integration (#33621)
DataDog/documentation: content/en/security/ticketing_integrations.md

Reviewer: This part is confusing since there is no explicit way for users to delete cases.

Reviewer: We could add some more information:
- detaching a case/Jira issue does not delete it
- if there are no findings left in the ticket, it will be closed (same thing for resolution/mute)
- and vice-versa for the auto-opening

Author: information added

## Case management and security products

Case Management is supported for all security products that use signals or findings:

- Code Security (in [Findings][5])
- Cloud Security (in [Findings][11])
- Cloud SIEM (in [Signals][4])
- App and API Protection (in [Signals][6] and [Findings][12])
- Workload Protection (in [Signals][7] and [Findings][13])

Open any signal or finding in these products or do a bulk selection of findings in the explorers, and use the **Create Ticket** button to create a case in Datadog.

## Bidirectional ticket syncing with Jira

Bidirectional syncing enables you to update Jira tickets automatically when changes occur in Datadog, and update some Datadog information when changes occur in Jira.

### Supported products

Bidirectional syncing is supported for the following Code and Cloud Security finding categories:

- Libraries (SCA)
- Static Code (SAST)
- Runtime Code (IAST)
- Secret Scanning (SDS)
- Infrastructure as Code (IaC)
- Misconfigurations
- Identity Risks
- Host and Container Vulnerabilities
- App and API Protection

> - Workload Protection

Reviewer: It's only true for their findings based explorers. Their signals explorers only offers non bi-synced CM tickets

Author: yes i specified finding categories on line 55 above

   4. Click the **Jira** tab.
   5. Verify that the **Case Management  <-> Jira Integration** section exists and bidirectional sync is enabled.

{{< img src="security/jira_modal.png" alt="Modal used to create a Jira ticket for a Security finding, with bidirectional sync enabled." responsive="true" style="width:50%;">}}

You are ready to start creating bidirectional Case Management tickets.

If you do not see the **Case Management  <-> Jira Integration** section, ensure that you have completed the prerequisites.

### Create bidirectional tickets

The following steps create a bidirectional ticket for a Security finding.

1. Open [any product supporting bidirectional ticket syncing][20].
2. Open any Security finding.
3. Locate the **Create Ticket** option. The option is available in **Next Steps** or **Repositories** (in **Libraries (SCA)**). The button will open a **Create ticket** modal.
4. Create ticket for any third party tool supported (see sections below)

{{% collapse-content title="Jira ticket" level="h4" expanded=false %}}

> 1. Click the **Jira** tab. You can use a new or existing ticket. Let's look at creating new Jira ticket.

Reviewer 2: ```suggestion
1. Click the **Jira** tab. You can use a new or existing ticket. Let's look at creating a new Jira ticket.
```

### Create bidirectional Case Management tickets

The following steps create a bidirectional Case Management ticket for a Code Security finding.

1. Open Code Security [findings][5].
2. Open any finding.
3. Locate the **Create Ticket** option. The option is available in **Next Steps** or **Repositories** (in **Libraries (SCA)**).
4. Click the **Jira** tab. You can use a new or existing ticket. Let's look at creating new Jira ticket.

> 5. In **Sync with Datadog (via Case Management)**, complete the following settings:

Reviewer: The **Sync with Datadog (via Case Management)** text is not in the modal anymore. iirc it was replaced with this and the rest of the config is done by clicking **Manage**

<img width="776" height="322" alt="Image" src="https://github.com/user-attachments/assets/3178926c-38a0-4e7f-8b4b-4cca82e89c4b" />

> 5. In **Sync with Datadog (via Case Management)**, complete the following settings:

Author: good idea, done

In **Datadog Associated Case**, the related Datadog case is provided. Click the case name to open it in [Case Management][1].

#### Automatic detachment and ticket opening/closing

Archiving a case does not delete related Jira tickets, but deleting a case project detaches all tickets from related Security findings.

Detaching a ticket from a Security finding does not delete it.

> If there are no open findings left attached to a ticket (because they are all detached or resolved or muted), it will be automatically closed.

Reviewer 2: ```suggestion
If there are no open findings left attached to a ticket (because they are all detached or resolved or muted), it is automatically closed.
```
